packages/icon/src/react/icon-transformer.js

```diff
@@ -12,6 +13,15 @@ const pascalize = str => {
   return trimmed.substr(0, 1).toUpperCase() + camelize(trimmed.substr(1))
 }
 
+const insertAfterChar = (str, char, content) => {
+  const charIndex = str.indexOf(char)
+
+  if (charIndex === -1) return str
+
+  const offset = charIndex + 1
+  return [str.slice(0, offset), content, str.slice(offset)].join('')
```

Comment:
making an array to then concat strings could be simplified by just concating the strings.

Reply:
good catch. fixed